spec/requests/headers_spec.rb

```diff
@@ -19,6 +19,12 @@
     expect(response.code.to_i).to eq(404)
   end
 
+  it 'does not blow up with missing view templates' do
+    get sign_up_email_path, headers: { 'Accept' => "nessus=bad_bad_value'" }
```

Comment:
this appears to be a test related to a specific controller and not across the whole site -- does this reproduce on the controller spec for the registrations controller?

Reply:
Good point. I will move this to a controller spec.

Reply:
Done. PTAL.